docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups.md

```diff
@@ -51,11 +32,13 @@ Before scheduling a backup of your universe YCQL data, create a policy, as follo
 
 1. Provide the backup policy name.
 
-1. Ensure that the API type is set as YCQL.
+1. Select the API type for the backup.
+
+1. Select the backup storage configuration. The available configurations depend on your existing backup storage configurations. For more information, see [Configure backup storage](../configure-backup-storage/).
 
-1. Select the database to back up.
+1. Select the database/keyspace to back up.
 
-1. Specify whether you want to back up all tables in the keyspace to which the database belongs or only certain tables. If you choose **Select a subset of tables**, a **Select Tables** dialog opens allowing you to select one or more tables to back up.
+1. For YCQL backups, you can choose to back up all tables in the keyspace to which the database belongs or only certain tables. If you choose **Select a subset of tables**, a **Select Tables** dialog opens allowing you to select one or more tables to back up. When finished, click **Confirm**.
```

Comment:
On line 57 , there is a mention of parallel threads under bullet 11 which is not showing up in the UI.